int8 right_arm
float64 bowl_x_pos_torso_ll_frame
float64 bowl_y_pos_torso_ll_frame
float64 bowl_z_pos_torso_ll_frame
float64 bowl_radius
float64 pre_mix_height_above_bowl_z_pos
int8 is_mix
int8 lower_spoon
int8 do_circular_mix
int8 do_linear_mix
int8 do_whisk_mix
int8 raise_spoon
int8 circular_laps
int8 linear_laps
int8 whisk_laps
float64 circular_period_sec
float64 linear_period_sec
float64 whisk_period_sec
float64 whisk_ang_vel
float64 angular_force_mag
float64 down_force_mag
float64 linear_force_mag
float64 radial_stiffness 
float64 rotational_stiffness
float64 radial_pad
int8 is_angular_force 
int8 is_down_force
---
int8 status
